portal/models/intervention_strategies.py

```diff
@@ -230,89 +231,91 @@ def intro_html(assessment_status):
                 due_date = assessment_status.next_available_due_date(
                     classification='baseline')
                 assert due_date
+                h2 = _("Hi {}").format(user.display_name)
```

Comment:
I don't like the `h2` and `h4` variable names - too likely to age poorly, like when the format changes.

how about `welcome_text` and `reminder` or whatever suits you?